driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/Fixture.java

```diff
@@ -39,11 +40,8 @@
  * Helper class for asynchronous tests.
  */
 public final class Fixture {
-    public static final String DEFAULT_URI = "mongodb://localhost:27017";
-    public static final String MONGODB_URI_SYSTEM_PROPERTY_NAME = "org.mongodb.test.uri";
     private static final String DEFAULT_DATABASE_NAME = "JavaDriverReactiveTest";
```

Comment:
Is this still needed - or can we just get it from ClusterFixture? Not sure it needs to be different to sync tests.